src/util.c

```diff
@@ -740,6 +745,65 @@ get_random_bytes(uint num) {
 	return buf;
 }
 
+// Legacy naming scheme for virtio block devices
+// Reference : https://github.com/torvalds/linux/blob/master/drivers/block/virtio_blk.c#L478
```

Comment:
This specific file happens to not have any explicit copyright owners, so we should not add any attribution to our file.
For people to check who wrote this code, please replace that comment with:
```C
// Reference: linux/blob/master/drivers/block/virtio_blk.c @ <SHA1>
```
From there a git log will tell who wrote the code before we copied it. Additionally, potential future copyright owners will not be able to claim ownership of this code.